Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -37,8 +37,6 @@ public class RepositoryPackageContainerProvider: PackageContainerProvider {
 
     public func getContainer(for identifier: RepositorySpecifier) throws -> Container {
```

Comment:
Ok the duplicated code can be avoided using this https://github.com/apple/swift-package-manager/pull/851